components/org.wso2.bfsi.consent.management.common/src/main/java/org/wso2/bfsi/consent/management/common/caching/ConsentManagementBaseCacheKey.java

```diff
@@ -0,0 +1,38 @@
+/**
+ * Copyright (c) 2024, WSO2 LLC. (https://www.wso2.com).
+ * <p>
+ * WSO2 LLC. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ * <p>
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ * <p>
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.bfsi.consent.management.common.caching;
+
+/**
+ * Abstract class for Consent Management Cache Key.
+ */
+public class ConsentManagementBaseCacheKey {
+
+    public ConsentManagementBaseCacheKey() {
+
+    }
+
+    public ConsentManagementBaseCacheKey(String cacheKey) {
+
+    }
```

Comment:
Why these constructors are empty?

Reply:
Fixed in https://github.com/wso2-extensions/bfsi-consent-management/pull/5/commits/4d9138c332549a29ab817615c49fd31c6d8cc121